applications/nrf5340_audio/src/modules/hw_codec.h

```diff
@@ -21,6 +21,16 @@
  */
 int hw_codec_volume_set(uint8_t set_val);
 
+/**
```

Comment:
No changes to this file should be done in this PR?

Reply:
This has been discussed around the office, and while this value is probably at this point, it has been agreed that a `get` function here is a fair addition to the `hw_coded` module